### PR TITLE
Add random to Validate Class and Test

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | We have to use 3 overrides to use SortOrder::random() with this PR only 2. Later there will be PRs for all controllers with BCs for 9.0.0.
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Related PRs       | https://github.com/PrestaShop/PrestaShop/pull/24408 https://github.com/PrestaShop/PrestaShop/pull/28807 https://friends-of-presta.slack.com/archives/CKJS3QZHR/p1655798664951229
| How to test?      | no test
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

**Comment by ghost** [comment]
Sorry @prestonBot

File: classes/Validate.php [hunk 1]
```diff
@@ -811,7 +811,9 @@ public static function isZipCodeFormat($zip_code)
      */
     public static function isOrderWay($way)
     {
-        return !empty($way) && in_array(strtoupper($way), ['ASC', 'DESC']);
+        $lowerWay = strtolower($way);
+
+        return $lowerWay === 'asc' | $lowerWay === 'desc' | $lowerWay === 'random';
```
Comment (Progi1984) [inline classes/Validate.php]:
```suggestion
        return $lowerWay === 'asc' || $lowerWay === 'desc' || $lowerWay === 'random';
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: classes/Validate.php (hunk 1 above)
Comment (PululuK) [reply classes/Validate.php]: I suggest to use `in_array`

**Review by davidglezz — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-functional, what about the testing part in this one? automated tests?

**Review by florine2623 — APPROVED** [review]
Hello @okom3pom , @kpodemski ,

As discussed with @Robin-Fischer-PS automated tests will do the job.

So, after running the automated tests, no error has been found. It is QA ✅ 

Thanks!

**Comment by Progi1984** [comment]
Thanks @okom3pom & @florine2623

**Pull request merged**